common/lib/api-client/models.js

```diff
@@ -177,4 +177,19 @@ module.exports = {
       filesize: '',
     },
   },
+  allocation: {
```

Comment:
It would be good to extend the [models tests](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/lib/api-client/models.test.js) to includes tests for this new model.